redesign/background.js

```diff
@@ -29,3 +29,19 @@ chrome.action.onClicked.addListener(async (tab) => {
     files: ['popup.js']
   });
 });
+
+chrome.runtime.onMessage.addListener((message, sender, sendResponse) => {
+  if (message.type === 'archivebox_add') {
+    const response = fetch(`${message.url}/api/v1/cli/add`, {
```

Comment:
Instead of getting the `server_url` & `api_key` from the message, lets just get them here directly:

```suggestion
    const { archivebox_server_url, archivebox_api_key } = await chrome.storage.local.get([
        'archivebox_server_url',
        'archivebox_api_key'
    ]);
    const response = fetch(`${archivebox_server_url}/api/v1/cli/add`, {
```